add platforms

danielnachun opened the pull request.

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found some lint.

Here's what I've got...


For **recipe/meta.yaml**:

 * ❌ The ``conda-forge.yml`` file is not allowed to have duplicate keys.

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

danielnachun committed:
enable platforms

danielnachun committed:
MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.12.11.16.09.59

Other tools:
- conda-build 25.11.0
- rattler-build 0.51.0
- rattler-build-conda-compat 1.4.10
